server/httpSearch.js

```diff
@@ -21,6 +27,26 @@ const createService = (inventoryStore) => {
       .catch(console.error);
   });
 
+  service.get('/search/:market/:checkin/:checkout', (req, res) => {
+    const { market } = req.params;
+    const { firstNight, lastNight } = getStayBookendNights(req.params);
+    inventoryStore.getAvailableListings(market, firstNight, lastNight)
+      .then((listings) => {
```

Comment:
How long are these queries taking for you and how many listings do you have currently up? Try to see if these are still efficient at greater scale (10M+), the delay for one of these queries for me is about 15 seconds and it looks like you are fetching these from Postgres.

Reply:
15 seconds was considered "long" for Analysis on the back-end, it'll be very long for a Search Service.

Reply:
This was the query I was talking about in the check-in with Danny yesterday. When I first ran it against my dev db (> 236K listings and >34M availability records), it took 6-7s. After adding a couple of indexes, it's now around 1s. I'll probably try optimizing some more later.